### PR TITLE
Gradle update onSaveInstanceState and onRestoreInstanceState added

### PR DESCRIPTION
**Pull request by GuilhE**
Since `void setSelectedIndex(final int selectedIndex)` is a private method, I've added `onSaveInstanceState()` and `onRestoreInstanceState(final Parcelable parcel)`.
Even if `setSelectedIndex` becomes public, this state logic will spare us from the additional work ;)

### CODE REVIEW

**Comment by vivchar** [comment]
Hey, thank you for contribution, but looks like you changed the code style, please return it back and ping me

**Comment by GuilhE** [comment]
Changed Java code style indent to 8, instead of default 4, to match your code style

**Comment by vivchar** [comment]
Please see full changes, it does not help https://github.com/vivchar/ViewPagerIndicator/pull/3/files
That is in red anyway =(

File: viewpagerindicator/src/main/java/com/github/vivchar/viewpagerindicator/ViewPagerIndicator.java [hunk 1]
```diff
@@ -18,163 +19,176 @@
...
+                        if (mListener != null) {
+                                mListener.onPageScrollStateChanged(state);
+                        }
+                }
+        }
+
+        @Override
+        protected Parcelable onSaveInstanceState() {
+                final Bundle bundle = new Bundle();
+                bundle.putParcelable("SUPER", super.onSaveInstanceState());
```
Comment (vivchar) [inline viewpagerindicator/src/main/java/com/github/vivchar/viewpagerindicator/ViewPagerIndicator.java]: please use other key name, it is very simple

Comment (vivchar) [reply viewpagerindicator/src/main/java/com/github/vivchar/viewpagerindicator/ViewPagerIndicator.java]: and move to const

**Comment by GuilhE** [comment]
Nice catch! Changed ;)

File: viewpagerindicator/src/main/java/com/github/vivchar/viewpagerindicator/ViewPagerIndicator.java (hunk 1 above)
Comment (GuilhE) [reply viewpagerindicator/src/main/java/com/github/vivchar/viewpagerindicator/ViewPagerIndicator.java]: Nice catch! Changed ;)

**Comment by GuilhE** [comment]
Hello, any news?

**Comment by vivchar** [comment]
Hi, I will do that manually later, because your changes contains a lot of codestyle changes. I need only onSave and onRestoreMethods.

**Comment by GuilhE** [comment]
Ok.
I think you should be using the default configurations because if anyone want's to contribute you'll have the same problem over and over.
The default Android Studio's Java code style indent is 4, your code uses 8.

If you change that, all you have to do next is to open your files and **Control + Alt + L** | **Command + Option + L** and you're done.

Cheers.

**Pull request merged**